### PR TITLE
Make default ip / port values in docstrings match code and online doc

### PR DESCRIPTION
**Pull request by ogrisel**
Tiny PR to make examples in docstrings less surprising by using default values used elsewhere in the code and the online doc.

### CODE REVIEW

File: distributed/worker.py [hunk 1]
```diff
@@ -486,7 +486,7 @@ def compute_one(self, data, key=None, function=None, args=None, kwargs=None,
                     "Function: %s\n"
                     "args:     %s\n"
                     "kwargs:   %s\n",
-                    str(funcname(function))[:1000],
```
Comment (ogrisel) [inline distributed/worker.py]: Those trailing spaces were fixed by my editor. I can re-edit this commit to avoid trimming them if you prefer.

Comment (mrocklin) [inline distributed/worker.py]: The whitespace changes are fine with me

**Comment by mrocklin** [comment]
Failures are unrelated.  They're due to the latest Bokeh release.

**Comment by mrocklin** [comment]
I'll clean up the bokeh things sometime this weekend.

**Pull request merged**